solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBuilderBase.java

```diff
@@ -38,7 +38,8 @@ public abstract class HttpSolrClientBuilderBase<
   protected Set<String> urlParamNames;
   protected Integer maxConnectionsPerHost;
   protected ExecutorService executor;
-  protected boolean useHttp1_1 = Boolean.getBoolean("solr.http1");
+  protected final boolean defaultUseHttp1_1 = Boolean.getBoolean("solr.http1");
```

Comment:
this could be static (and thus moved above to statics); right?

Reply:
I don't think it should be static, because test cases almost certainly try to change it I imagine...